### DISCUSSION TITLE
Auth trigger breaks authentication

### DISCUSSION THREAD

**Discussion by namzug16** [supabase/supabase#21464] [Questions] [answered]
When creating a trigger on auth.users the authentication system breaks. I'm not able to create a new user and no feedback is given from the dashboard. 

This was not the case a couple of days ago though

To reproduce you can add a simple function and then add a trigger to it on the auth.users table

```
CREATE OR REPLACE FUNCTION public.handle_new_user()
 RETURNS trigger
 LANGUAGE plpgsql
 SECURITY DEFINER
AS $function$
begin
  insert into public.user (id, email)
  values (new.id, new.email);
  return new;
end;
$function$
;

CREATE TRIGGER on_auth_user_created AFTER INSERT ON auth.users FOR EACH ROW EXECUTE FUNCTION handle_new_user();
```

Because this table is private and as the dashboard says, it is managed by supabase, I'm not sure if this should be the expected behaviour or not, mainly because it was working before. 

I've tested this on a local instance with the cli and on an actual project.

**Comment by GaryAustin1** [comment, marked as answer]
You need to look in the postgres logs of the dashboard for the error with your function.
I'm moving this to discussion as your code works as is on a user table with id uuid type and email text type column.

**Comment by namzug16** [comment]
Yes, completely my bad xd, the table was called "users" and not "user". Thank you for the fast response tho :)
